src/extensions/default/HealthData/htmlContent/healthdata-notification-dialog.html

```diff
@@ -0,0 +1,16 @@
+<div class="health-data-notification modal">
```

Comment:
As @peterflynn was before (https://github.com/adobe/brackets/pull/10795#discussion_r27637175), I'm a concerned about the UX of a modal dialog too, especially for first-time users.
Right after starting Brackets, they'll see a dialog.